erpnext/education/utils.py

```diff
@@ -361,15 +370,17 @@ def check_content_completion(content_name, content_type, enrollment_name):
 		return False
 
 def check_quiz_completion(quiz, enrollment_name):
-	attempts = frappe.get_all("Quiz Activity", filters={'enrollment': enrollment_name, 'quiz': quiz.name}, fields=["name", "activity_date", "score", "status"])
+	attempts = frappe.get_all("Quiz Activity", filters={'enrollment': enrollment_name, 'quiz': quiz.name}, fields=["name", "activity_date", "score", "status", "time_taken"])
```

Comment:
Formatting to improve readability?

```suggestion
	attempts = frappe.get_all('Quiz Activity',
		filters={
			'enrollment': enrollment_name,
			'quiz': quiz.name
		},
		fields=['name', 'activity_date', 'score', 'status', 'time_taken']
	)
```